rpcs3/Emu/Cell/lv2/sys_ppu_thread.cpp

```diff
@@ -47,6 +47,14 @@ bool ppu_thread_exit(ppu_thread& ppu)
 		dct.used -= ppu.stack_size;
 	}
 
+	std::string str = fmt::format("%s", ppu.call_history);
```

Comment:
I would prefer to avoid unnecessarily calling debug-related code if the user has not enabled it. This is more for regression safety than anything else, bugs in the history mechanism should have no impact on the general user.

Reply:
Fixed, only enabled path when setting is enabled. (index must be 0 if not)